efficientdet/keras/efficientdet_keras_test.py

```diff
@@ -66,20 +66,28 @@ def test_model_output(self):
       sess.run(tf.global_variables_initializer())
       legacy_class_out, legacy_box_out = sess.run(feats)
     for i in range(3, 8):
-      self.assertAllClose(
-          keras_class_out[i - 3], legacy_class_out[i], rtol=1e-4, atol=1e-4)
-      self.assertAllClose(
-          keras_box_out[i - 3], legacy_box_out[i], rtol=1e-4, atol=1e-4)
+      self.assertAllClose(keras_class_out[i - 3],
+                          legacy_class_out[i],
+                          rtol=1e-4,
+                          atol=1e-4)
+      self.assertAllClose(keras_box_out[i - 3],
+                          legacy_box_out[i],
+                          rtol=1e-4,
+                          atol=1e-4)
 
     feats = tf.ones(inputs_shape)
     model = efficientdet_keras.EfficientDetNet(config=config)
     model.load_weights(tmp_ckpt)
     eager_class_out, eager_box_out = model(feats)
     for i in range(3, 8):
-      self.assertAllClose(
-        eager_class_out[i - 3], legacy_class_out[i], rtol=1e-4, atol=1e-4)
-      self.assertAllClose(
-        eager_box_out[i - 3], legacy_box_out[i], rtol=1e-4, atol=1e-4)
+      self.assertAllClose(eager_class_out[i - 3],
```

Comment:
I realized our formater difference is caused by the yapf version difference.

You probably use the stable yapf-0.30, but the internal version is based on the latest yapf. They are mostly the same with a few small differences.

Could you add an option "SPLIT_BEFORE_EXPRESSION_AFTER_OPENING_PAREN = True" in your yapf config and rerun your auto-format for this PR? (same as the latest yapf code: https://github.com/google/yapf/blob/51ffe2d07930a509ecb2ef454a7b251eeeac0a59/yapf/yapflib/style.py#L502)